test/Libraries/RevitNodesTests/Elements/ElementTests.cs

```diff
@@ -436,5 +436,42 @@ private static void AssertElementsAreJoined(Element element, Element otherElemen
         }
 
         #endregion
+
+        [Test]
+        [TestModel(@".\Element\elementIntersection.rvt")]
+        public void CanSuccessfullyGetIntersectingElementsOfSpeceficCategory()
+        {
+            // Element to check intersections on
+            int intersectionElementId = 316167;
+            var intersectionElement = ElementSelector.ByElementId(intersectionElementId, true);
+
+            // Element intersecting
+            int structuralFramingId = 316318;
+            var structuralFramingElement = ElementSelector.ByElementId(structuralFramingId, true);
+            int floorId = 316539;
+            var floorElement = ElementSelector.ByElementId(floorId, true);
+            int wallId = 316246;
+            var wallElement = ElementSelector.ByElementId(wallId, true);
+
+            // Get intersecting elements of category
+            Revit.Elements.Category structuralFrameCategory = Revit.Elements.Category.ByName("StructuralFraming");
```

Comment:
use `var` instead of full type names, it's distracting how long they are and we already have `Category` in the variable name.